fabfile.py

```diff
@@ -396,7 +386,7 @@ def _kernel_deps():
         'xmlto',
         'libgtk2.0-dev',
 	'libunwind-setjmp0-dev',
-	'libunwind7-dev',
+#	'libunwind7-dev',
```

Comment:
we should try not to break things we're still using today, yeah.  

maybe it's obsolete because we no longer build perftools but rather install it?  I guess a reasonable test would be to uninstall it on a trusty build and see that the build still works